### ISSUE TITLE
async ready version of it

### ISSUE THREAD

**Issue by JaredAAT** [xabikos/vscode-jasmine#21]
It would be great if there was an async version of `it`, that the auto complete would add `it('should behave...', async () => {})`

**Comment by jjspace** [comment]
I came to add a PR for this exact thing but it seems this repo is pretty stale. Regardless, you can [Create your own snippets](https://code.visualstudio.com/docs/editor/userdefinedsnippets#_create-your-own-snippets) and add the snippet below to achieve what you're after

```json
"it async": {
	"prefix": "ita",
	"description": "creates a test method with an async body",
	"body": "it('${1:should behave...}', async () => {\n\t$2\n});"
}
```

**Comment by JaredAAT** [comment]
@jjspace yep, i forked this to do exactly that.  I might try and add some of the other PR's thata re open and look at the suggestions in the issues to add to the repo i forked.
